sw/host/opentitanlib/src/transport/ti50emulator/emu.rs

```diff
@@ -39,6 +39,8 @@ pub struct EmulatorProcess {
     state: EmuState,
     /// Handle to Emulator sub-proccess.
     proc: Option<Child>,
+    /// Counter of 'power' cycle
+    pcount: u32,
```

Comment:
maybe a more descriptive name, e.g.:
```suggestion
    power_cycle_count: u32,
```